Samples/auth/Outlook-Add-in-SSO/AttachmentDemoWeb/Functions/FunctionFile.js

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+Office.initialize = function () {
+
+}
+
+async function saveAllAttachments(event) {
+    var attachmentIds = [];
+    let accessToken = await OfficeRuntime.auth.getAccessToken({ allowSignInPrompt: true, allowConsentPrompt: true, forMSGraphAccess: true });
```

Comment:
There's no error handling for the call to getAccessToken and no branching to the fallback method when the call fails.